glide.lock

```diff
@@ -192,9 +192,11 @@ imports:
   - transport/zipkin
   - utils
 - name: github.com/uber/jaeger-lib
-  version: 7f95f4f7e80028096410abddaae2556e4c61b59f
+  version: 4267858c0679cd4e47cefed8d7f70fd386cfb567
   subpackages:
   - metrics
+  - metrics/adapters
+  - metrics/expvar
   - metrics/go-kit
   - metrics/go-kit/expvar
```

Comment:
weird, even if I remove this line/pkg, glide adds it back, even though all code using it seems to be gone.